model-server/src/test/kotlin/org/modelix/model/server/ReplicatedRepositoryTest.kt

```diff
@@ -116,9 +115,27 @@ class ReplicatedRepositoryTest {
         }
     }
 
-    @Ignore
-    @Test
-    fun `concurrent write`() = runTest {
+    @Test fun `concurrent write 0`() = concurrentWrite(0)
+
+    @Test fun `concurrent write 1`() = concurrentWrite(1)
+
+    @Test fun `concurrent write 2`() = concurrentWrite(2)
+
+    @Test fun `concurrent write 3`() = concurrentWrite(3)
+
+    @Test fun `concurrent write 4`() = concurrentWrite(4)
+
+    @Test fun `concurrent write 5`() = concurrentWrite(5)
+
+    @Test fun `concurrent write 6`() = concurrentWrite(6)
+
+    @Test fun `concurrent write 7`() = concurrentWrite(7)
+
+    @Test fun `concurrent write 8`() = concurrentWrite(8)
+
+    @Test fun `concurrent write 9`() = concurrentWrite(9)
+
+    fun concurrentWrite(iteration: Int) = runTest {
```

Comment:
This setup can be simplified with [`@RepeatedTest`](https://www.baeldung.com/junit-5-repeated-test#accessing-the-repetitioninfo).